types/yog-ral/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for yog-ral 0.20
+// Project: https://gitlab.baidu.com/fex/node-ral
+// Definitions by: ssddi456 <https://github.com/ssddi456>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7

```

Comment:
This restricts the library to only be used with TypeScript 3.7+. This file doesn't use any 3.7 specific features, so to allow more users to use this package, this can be bumped down.

Reply:
ok